src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrapGh.hpp

```diff
@@ -0,0 +1,191 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"  // for tags
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+// IWYU pragma: no_forward_declare Tags::deriv
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace GeneralizedHarmonic {
+namespace Solutions {
+
+/*!
+ * \brief A wrapper for general-relativity analytic solutions that loads
+ * the analytic solution and then adds a function that returns
+ * any combination of the generalized-harmonic evolution variables,
+ * specifically `gr::Tags::SpacetimeMetric`, `GeneralizedHarmonic::Tags::Pi`,
+ * and `GeneralizedHarmonic::Tags::Phi`
+ */
+template <typename SolutionType>
+class WrapGh : public SolutionType {
```

Comment:
I'm confused by the name. This is a `Gh` solution, it wraps a `GR` solution, so why not `WrapGr` (or better yet `WrappedGr`)?